cspell.json

```
@@ -0,0 +1,67 @@
+{
+  "allowCompoundWords": true,
+  "caseSensitive": false,
+  "language": "en,ru,uk",
+  "minWordLength": 3,
+  "dictionaryDefinitions": [
+    {
+      "addWords": true,
+      "name": "custom-dict",
+      "path": "cspell/custom-dict.txt"
+    }
+  ],
+  "dictionaries": [
+    "aws",
+    "bash",
+    "companies",
+    "cpp",
+    "css",
+    "custom-dict",
+    "data-science",
+    "docker",
+    "dotnet",
+    "en_us",
+    "en-common-misspelling",
+    "fonts",
+    "fullstack",
+    "golang",
+    "html",
+    "k8s",
+    "makefile",
+    "misc",
+    "npm",
+    "people-names",
+    "python",
+    "ru_RU",
+    "rust",
+    "software-terms",
+    "uk_UA",
+    "win32"
+  ],
+  "enabled": true,
+  "files": ["docs/**/*.{example,md}"],
```

```suggestion
  "files": ["docs/**/*.md"],
```

I see no need in checking examples, as they contain go code. Do I miss something?

---

Done.

This is a leftover from [my first spellchecking PR](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6362/files#diff-82d4d91bd662440ab351943c532fc28affb11b181476485c34ea9e7325cf3dac) -- these files are under `docs` and also had some typos.